src/main/webapp/auth.js

```diff
@@ -17,7 +17,7 @@
 // TODO: Manage User Sign In State (Issue #13)
 
 /* eslint-disable no-unused-vars */
-/* global gapi */
+/* global gapi, addCookie */
```

Comment:
Don't need to worry about this for this PR: this is one of the main reasons why I like the `require` approach: it reduces the need to rely on global values

Reply:
I created an issue for this #31 .

I tried to use require, but it was throwing errors since it's "unrecognized". Turns out I have to install CommonJS/find a way to use it. 

I agree with you - this is messy